Tidyver_Help.Rmd

```diff
@@ -0,0 +1,310 @@
+#  TidyVerse
+
+Zhisen Cai
+
+Sometimes we may forget the way to use Tidyverse. Therefore, I create this for help. People can finish most of the basic part of tidyverse by using this. I separate it into 4 parts, manipulation between two datasets, manipulation on rows, manipulation on columns and group_by.  With the examples of each function, people can know how to use the function and how the function works.
+
+```{r}
+library(tidyverse)
+```
+Magrittr use pipeline to convey the data. (%>%) We have to use a dot to represent the thing we convey if it is not used in the first parameter. 
+```{r}
+# we can omit '.'
+c(1, 3, 4, 5, NA) %>% mean(., na.rm=TRUE)
+c(1, 3, 4, 5, NA) %>% mean(na.rm = TRUE)  
+# we can not omit '.' in the second parameter
+c(1, 3, 4, 5) %>% plot(., main=paste(., collapse=", "))
+c(1, 3, 4, 5) %>% plot(main=paste(., collapse=", "))
+```
+
+### 0. Connection Bewtween two datasets
```

Comment:
Please remove the numbers here and all below, use only multiple hashtags.
```suggestion
### Connection Bewtween two datasets

...

### Combine two datasets by value: join

...
```

Reply:
I commit the second edition. I'm really sorry for missing this conversation.

Reply:
I saw you fixed the hashtags! Could you also change your filename to all lowercases (ex. tidyver_help.Rmd)? Thanks!